java/code/src/com/redhat/rhn/frontend/xmlrpc/user/UserHandler.java

```diff
@@ -1141,4 +1144,63 @@ public int setErrataNotifications(User loggedInUser, String login, Boolean value
         targetUser.setEmailNotify(BooleanUtils.toIntegerObject(value));
         return 1;
     }
+
+    /**
+     * @param user The current user
+     * @param unread Unread notifications
+     * @return Returns a list of notifications
+     * @apidoc.doc Get all notifications from a user.
+     * @apidoc.param #session_key()
+     * @apidoc.param #param_desc("boolean", "unread", "Read notifications.")
+     * @apidoc.returntype #return_notifications()
```

Comment:
You must tell the API docs generator to print out the structure of your serializer like in this example:

https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/frontend/xmlrpc/user/UserHandler.java#L109-L112